### PR TITLE
Fix manual docker command output for Python.

### LINKED ISSUE #13230: Python v1.0.x interop matrix failure
```
stdin: is not a tty
running run_interop
Traceback (most recent call last):
  File "src/python/grpcio_tests/setup.py", line 115, in <module>
    test_runner=TEST_RUNNER,
  File "/usr/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/usr/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/var/local/git/grpc/src/python/grpcio_tests/commands.py", line 201, in run
    self.run_client()
  File "/var/local/git/grpc/src/python/grpcio_tests/commands.py", line 217, in run_client
    client.test_interoperability()
  File "/var/local/git/grpc/src/python/grpcio_tests/tests/interop/client.py", line 123, in test_interoperability
    test_case = _test_case_from_arg(args.test_case)
  File "/var/local/git/grpc/src/python/grpcio_tests/tests/interop/client.py", line 117, in _test_case_from_arg
    raise ValueError('No test case "%s"!' % test_case_arg)
ValueError: No test case "None"!
FAILED: python__python_v1.0.x:python:grpc-test4:cancel_after_first_response" [ret=1, pid=48163, time=2.8sec]
```

### PR DESCRIPTION
**Pull request by adelez**
Need to escape " in the Python client command when saving it to the manual log.
This also fixes #13230.

### CODE REVIEW

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

File: tools/interop_matrix/client_matrix.py [hunk 1]
```diff
@@ -64,14 +64,14 @@ def get_github_repo(lang):
...
-    #    'v1.4.2',
-    #    'v1.6.6',
-    #],
+    'python': [
+        'v1.0.x',  #Fail to run the test. #13230.
+        'v1.1.4',
+        'v1.2.5',
+        'v1.3.9',
+        'v1.4.2',
+        'v1.6.6',
```
Comment (nathanielmanistaatgoogle) [inline tools/interop_matrix/client_matrix.py]: Any desire to add a `v1.7.x` tag to this list?

File: tools/interop_matrix/client_matrix.py [hunk 2]
```diff
@@ -64,14 +64,14 @@ def get_github_repo(lang):
...
-    #'python': [
-        #'v1.0.x',  #Fail to run the test. #13230.
-    #    'v1.1.4',
-    #    'v1.2.5',
-    #    'v1.3.9',
-    #    'v1.4.2',
-    #    'v1.6.6',
-    #],
+    'python': [
+        'v1.0.x',  #Fail to run the test. #13230.
```
Comment (nathanielmanistaatgoogle) [inline tools/interop_matrix/client_matrix.py]: Is #13230 still an active issue? (@kpayson64?)

File: tools/interop_matrix/testcases/python__master [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+echo "Testing ${docker_image:=grpc_interop_python:797ca293-94e8-48d4-92e9-a4d52fcfcca9}"
```
Comment (nathanielmanistaatgoogle) [inline tools/interop_matrix/testcases/python__master]: It's surprising to me that fixing this problem involves adding this whole large file to the codebase - was this an inadvertent addition or does this need to be here?

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

File: tools/interop_matrix/client_matrix.py (hunk 1 above)
Comment (adelez) [reply tools/interop_matrix/client_matrix.py]: It'll be in #13398.

File: tools/interop_matrix/testcases/python__master (hunk 3 above)
Comment (adelez) [reply tools/interop_matrix/testcases/python__master]: This is the test commands file that's needed for every language by the framework. Previously I was just testing locally.

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

File: tools/interop_matrix/testcases/python__master (hunk 3 above)
Comment (nathanielmanistaatgoogle) [reply tools/interop_matrix/testcases/python__master]: Well... okay.

**Comment by adelez** [comment]
That echo should probably be removed. I'll do a cleanup in a separate PR as it's not Python specific.

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by adelez** [comment]
Asan C++ #13122
Basic Tests MacOS [dbg] #13381,  #13085
Basic Tests Multi-language Linux #13085 
Basic Tests Windows [dbg] #12595

File: tools/interop_matrix/testcases/python__master (hunk 3 above)
Comment (adelez) [reply tools/interop_matrix/testcases/python__master]: This is the current design. Here's the background: we debated on whether to create separate file for test cases for bake them into the images. Folks voted for the first, hence the need of these files checked in. But as I took over this project and started expansion, I felt the 2nd option probably works better. But it'll require a major architectural change.

**Pull request merged**